vmm/src/memory_manager.rs

```diff
@@ -185,6 +187,9 @@ pub struct MemoryManager {
     guest_ram_mappings: Vec<GuestRamMapping>,
 
     pub acpi_address: Option<GuestAddress>,
+    #[cfg(target_arch = "aarch64")]
+    // Flash device for UEFI on AArch64
```

Comment:
I don't think you need this comment. It says nothing more than the code.

Reply:
Ok